tests/pipeline_test.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
```

Comment:
Does this even run without an FPGA SDK? If not, it will try to run automatically on CI, and probably fail.

Reply:
Renamed